<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptchaImageDialog</class>
 <widget class="QDialog" name="CaptchaImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Captcha - YzTelecomAIOCP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_CaptchaImage">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_CaptchaCode"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
